Sources/PklSwift/EvaluatorOptions.swift

```diff
@@ -191,7 +191,7 @@ extension EvaluatorOptions {
         return .init(
             packageUri: nil,
             type: "project",
-            projectFileUri: "file://\(projectBaseURI)/PklProject",
+            projectFileUri: "\(projectBaseURI.appending(component: "PklProject"))",
```

Comment:
A workaround in swift 5.9.2 is to use `projectBaseURI.appendingPathComponent("PklProject")`. Just tested in a dev container